### DISCUSSION TITLE
Suggestion: Table of contents by category

### DISCUSSION THREAD

**Discussion by ghost** [tteck/Proxmox#547] [Ideas]
Good evening,

As the number of scripts grow, I find it harder to navigate, and to find related selfhosted solutions. An idea to fix this would be to add a table of contents and regroup scripts by larger category, see the [awesome self-hosted list on GitHub](https://github.com/awesome-selfhosted/awesome-selfhosted#table-of-contents) as an example.

Thanks again for all the work,

**Comment by tteck** [comment]
Would this be better?

https://github.com/tteck/Proxmox/blob/main/README.md

**Reply by ghost** [reply]
Yes! That's exactly what I had in mind
